x-pack/plugins/ingest_pipelines/public/application/mount_management_section.ts

```diff
@@ -28,6 +28,7 @@ export async function mountManagementSection(
     metric: uiMetricService,
     documentation: documentationService,
     api: apiService,
+    notifications: coreSetup.notifications,
```

Comment:
nit: `NotificationsStart` should be available on `KibanaContextProvider`

Reply:
I remember I made a similar comment on one of your previous PRs, but I think I was incorrect. It actually looks like this is not possible with `KibanaContextProvider`, only if you create your own context with `createKibanaReactContext`. I'm going to leave it as is for now.

Reply:
Right, thanks for clarifying!